### ISSUE TITLE
'Recently used' is displayed even when history is disabled

### ISSUE THREAD

**Issue by cmburg** [elementary/files#1167]
## Prerequisites
- [x] I have searched open and closed issues for duplicates.

## Describe the bug
The "recently used" item appears in the tree no matter what the privacy setting is doing.

### To Reproduce
1. Open files and click "recent files" in the left menu tree. You should see some files.
2. Go to 'apps'->'System settings'->'privacy'->'disable'
3. You should see the ffolder is refreshed and displays as empty.
4. Despite that the item "recently used" is still shown even it has no use.

### Expected behavior
The item disappears when history is disabled.

## Platform Information
![Screenshot von 2020-01-16 20 27 48](https://user-images.githubusercontent.com/18635557/72668095-64f0f200-3a23-11ea-8a4a-753028c0758b.png)

Please check what applies:
-->
- [ ] I'm using the latest version from git that I've manually compiled
- [x] I'm using the latest released stable version

**Comment by jeremypw** [comment]
You should see a special page when you press "Recent" with history off, not an empty folder.  The "PrivacyOn" page explains why the folder is empty and gives a link to the privacy settings.  I am not sure we want to lose this.  Flagging for UX team input.

**Comment by cmburg** [comment]
> You should see a special page when you press "Recent" with history off, not an empty folder. The "PrivacyOn" page explains why the folder is empty and gives a link to the privacy settings. I am not sure we want to lose this. Flagging for UX team input.

You are right. I had the settings window open and it hid the message. Despite that I have the opinion that the element should not be shown in the tree when history is disabled.

**Comment by jeremypw** [comment]
Closing as "Out of Scope" - intended behaviour
